docs/cluster-management/configure-api.md

```diff
@@ -665,6 +668,46 @@ release:
     headerValue: stable
 ```
 
+### Configure Redis Lock
```

Comment:
can you add that, `redis lock` is used to ensure that the same join job build is updated sequentially to ensure no data loss